### PR TITLE
Add ErrorBoxFixer from KZTimer

### PR DESCRIPTION
**Pull request by walliski**
These models are used in some maps, but are not packaged with the map
itself. This means that we need to provide them separately for the
client, else they will just see a huge error box.

The files that are added here, is a cleaned up version of the models and
stuff from KZTimer repository. I did not include all the files, as I
could not see them being used. It could however be that we are still
missing some files that are normally in KZTimer?

### CODE REVIEW

**Comment by walliski** [comment]
Open questions:

1. There were more models/sprites/materials etc. in KZTimer. I chose to not include them here, since I could not find them being used. Not sure if they are somehow used through some other binary files referencing them?

```
materials/models/props/startkztimer.vmt
materials/models/props/startkztimer.vtf
materials/models/props/stopkztimer.vmt
materials/models/props/stopkztimer.vtf
materials/models/props/switch.vmt
materials/models/props/switch.vtf
materials/models/props/switch001.vmt
materials/models/props/switch001.vtf
materials/models/props/switch001_exponent.vmt
materials/models/props/switch001_exponent.vtf
materials/models/props/switch001_lightwarp.vmt
materials/models/props/switch001_lightwarp.vtf
materials/models/props/switch001_normal.vmt
materials/sprites/bluelaser1.vtf
materials/sprites/halo01.vmt
materials/sprites/halo01.vtf
materials/sprites/laser.vmt
models/props/switch001.dx90.vtx
models/props/switch001.mdl
models/props/switch001.phy
models/props/switch001.vtx
models/props/switch001.vvd
```

2. Some files came with a .ztmp version also. Do we want to include these or not?

File: addons/sourcemod/scripting/gokz-errorboxfixer.sp [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#include <sourcemod>
+#include <sdktools>
+
+#include <gokz>
```
Comment (walliski) [inline addons/sourcemod/scripting/gokz-errorboxfixer.sp:4]: For updater URL and version only. Could this be moved to not required maybe? Not that I see why you would want this plugin without running a kz server? 🤔

Comment (Sikarii) [reply addons/sourcemod/scripting/gokz-errorboxfixer.sp:4]: The main include file does not add dependency to any GOKZ plugins, whether you have it under optional or not it does not matter.

**Comment by zealain** [comment]
I doubt those files will ever change, so I'd rather not bloat the release with the .ztmp files. And everything deployment related will be handled, don't worry about that. I think this looks fine, but @GameChaos anyway cause KZTimer ^^

**Comment by GameChaos** [comment]
from my googling, a .ztmp file is generated automatically so you don't need it.

**Comment by walliski** [comment]
Alright, removed the ztmp files.

**Pull request merged**